src/Service.Tests/Authentication/Helpers/WebHostBuilderHelper.cs

```diff
@@ -41,17 +41,24 @@ public static class WebHostBuilderHelper
         /// - DAB's Simulator/ EasyAuth authentication middleware and ClientRoleHeader middleware
         /// - dotnet's authorization middleware.
         /// </summary>
-        /// <param name="provider">Runtime configured identity provider name.</param>
+        /// <param name="provider">Runtime configured identity provider name. This is different that authentication scheme name because
```

Comment:
```suggestion
        /// <param name="provider">Runtime configured identity provider name. This is different than authentication scheme name because
```